### DISCUSSION TITLE
Publishing to repos

### DISCUSSION THREAD

**Discussion by codefish1** [bmTas/JRecord#15] [Ideas]
I'd love to help get this and https://github.com/bmTas/cb2xml building under actions and publishing to central maven repos so that its easier for people to make use of it.

Downloading JARs and placing them in your local repo doesn't make for a good user experience and breaks at a number of places so as CICD where it is expected to be pulled down

**Comment by bmTas** [comment]
It would certainly be good to get it into a central maven repo. I started looking at it but got interrupted by other things.
not sure I am able to devote the time to do it.

**Comment by codefish1** [comment]
I can help with that and might be able to get some people from work to help as well.  I'll start with some PRs to get it building under Github actions etc if your happy with that

**Comment by bmTas** [comment]
I would like that. I am busy at the moment (Death in the family).

Currently the Source Code started as a standard ant-eclipse project, maven was then written for it.

One thought I had was to  Create standard Eclipse-Maven (or Netbeans-maven) projects and rebuild the projects in a standard way. I have played around with this already.

---

Also I have some Cobolo2json upgrade request https://sourceforge.net/p/record-editor/discussion/467595/thread/3b9db9af9d/?limit=25#e792
